ansible/templates/conf/nomad/011-integrations.hcl.j2

```diff
@@ -6,5 +6,11 @@ consul {
 
 vault {
   enabled = true
-  token = "{{ lookup('env', 'vault_master_token') }}"
-}
\ No newline at end of file
+
+  # `token` value is injected as VAULT_TOKEN env variable for Nomad service
+  # Check `ansible/templates/nomad.service.j2` for details
+  # token = "{{ lookup('env', 'vault_master_token') }}" todo: remove this line
```

Comment:
Does this todo imply that this line should be removed or is it something else?

Reply:
It is implied that the token won't be set up directly here as clear text. The comment is provided

Reply:
What means: `todo: remove this line` ?

Reply:
I agree.